### ISSUE TITLE
Record links are inconsistently broken

### ISSUE THREAD

**Issue by ansell** [AtlasOfLivingAustralia/data-management#375]
The ANWC has said that previously there were some records linked together using a specific syntax. Need to research how this was done in the past and what to do to make it work again now.

Refs ticket 23624 in helpdesk

**Comment by ansell** [comment]
A pointer from a previous email is the following syntax example in a field named ``associatedOccurrences``:

```
associatedOccurrences
--
<a href=http://biocache.ala.org.au/occurrences/search?q=institution_code:ANWC%20AND%20collection_code:Amphibians%20AND%20catalogue_number:A01702>source specimen ANWC:Amphibians:A01702</a>
```

**Comment by elywallis** [comment]
ANWC (Margaret Cawsey) also asked me to help her with understanding this issue. Looking at NMV tissue records, sometimes the link is made correctly but sometimes it's not. The field in Biocache you want to look at is associatedOccurrences
Correct: https://biocache.ala.org.au/occurrences/df9c78e6-6908-4ae4-8b72-09b22ef9c9ff 
Failed: https://biocache.ala.org.au/occurrences/d9f3141a-9f79-48f6-9ba1-2f25104f20d6 

****Note: from this example the link fails to be created simply because of html syntax. The URL needs double quotes around it.** 

Could part of the fix be as simple as that?

**Comment by ansell** [comment]
@elywallis I have advised Margaret that there are no developer resources assigned to Biocache at this point and that this is the reason that her issues are not being fixed. This and #296 are both blocked on a developer being assigned to biocache-store to investigate this and other breakages since we switched to the new biocache. The switch to the clustered biocache was unfortunately done on a time basis so developers could be reassigned to work on other ALA projects, and that has caused the new biocache to not have the same capabilities as the previous biocache.
